index.js

```diff
@@ -14,8 +14,14 @@ module.exports = function (options) {
       }
       var msg = message.message
       switch (message.severity) {
-        case 1: file.warn(msg, pos); break
-        case 2: file.fail(msg, pos); break
+        case 1:
+          file.message(msg, pos)
+          break
+        case 2:
+          try {
+            file.fail(msg, pos)
+          } catch (err) {}
```

Comment:
What's the purpose of the unhandled try/catch? Does `file.fail(...)` throw an error the runner doesn't catch and handle itself?